lab_1_keywords_tfidf/main.py

```diff
@@ -66,8 +91,20 @@ def get_top_n(frequencies: dict[str, Union[int, float]], top: int) -> Optional[l
 
     In case of corrupt input arguments, None is returned
     """
-    pass
-
+    if not frequencies or not top:
+        return None
+    if not isinstance(frequencies, dict) or not type(top).__name__ == 'int':
```

Comment:
it is a very creative type check! 
however, it is more common to check that a value is integer and not boolean, as booleans are considered to also be integer, but not vice versa

Reply:
done